src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/InputCustomAnnotation.java

```diff
@@ -0,0 +1,83 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation; //indent:0 exp:0
+
+import java.lang.annotation.ElementType; //indent:0 exp:0
+import java.lang.annotation.Repeatable; //indent:0 exp:0
+import java.lang.annotation.Retention; //indent:0 exp:0
+import java.lang.annotation.RetentionPolicy; //indent:0 exp:0
+import java.lang.annotation.Target; //indent:0 exp:0
+
+public @interface InputCustomAnnotation { //indent:0 exp:0
+} //indent:0 exp:0
+
+     class TestClass { //indent:5 exp:0 warn
+String author; //indent:0 exp:4 warn
+} //indent:0 exp:0
+
+     interface TestInterface {} //indent:5 exp:0 warn
+
+     enum TestEnumInput {} //indent:5 exp:0 warn
```

Comment:
We can remove anything that isn't helping an annotation with it's test.
I just added these in issue to showcase we get violations other block types correctly.